spec/requests/user_pages_spec.rb

```diff
@@ -377,11 +377,11 @@
 
       it "displays the feedback with a 'Resume Feedback' action if not submitted" do
         visit feedbacks_user_path(reviewer)
-        page.should have_selector('#feedbacks td', text: ac.user.name)
-        page.should have_selector('#feedbacks td', text: review.review_type)
-        page.should have_selector('#feedbacks td', text: review.feedback_deadline.to_s(:short_date))
-        page.should have_selector('#feedbacks td', text: feedback.updated_at.to_s(:short_date))
-        page.should have_selector('#feedbacks td .fa-pencil')
+        page.should have_selector('#feedback-requests td', text: ac.user.name)

```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.